README.md

```diff
@@ -46,12 +43,12 @@ Options:
 
 Some additional tips:
 
-* _File paths_: all file paths are relative to current working directory (`process.cwd()`) or can be absolute.
-* `-a|--action`:
-    * `server`: Run a development in-memory web-server with hot reload for your presentation.
-    * `build`: Output a full production website static build to the `-o|--output` directory.
-* `-s|--src`: The slide source file path. If a file has a `.md` suffix is processed as straight Markdown. If a file has a `.mdx` suffix it is processed as enhanced MDX.
-* `-t|--theme`: A `require|import`-able file (e.g., `.js` or `.json`) that contains an object to override the Spectacle default theme in places.
+- _File paths_: all file paths are relative to current working directory (`process.cwd()`) or can be absolute.
+- `-a|--action`:
+  - `server`: Run a development in-memory web-server with hot reload for your presentation.
+  - `build`: Output a full production website static build to the `-o|--output` directory.
+- `-s|--src`: The slide source file path. If a file has a `.md` suffix is processed as straight Markdown. If a file has a `.mdx` suffix it is processed as enhanced MDX.
+- `-t|--theme`: A `require|import`-able file (e.g., `.js` or `.json`) that contains an object to override the Spectacle default theme in places.
 
```

Comment:
For some reason Prettier changed these, but I think the output is still the same.

<img width="691" alt="Screen Shot 2020-01-07 at 2 24 00 PM" src="https://user-images.githubusercontent.com/1738349/71926668-6ab02300-3159-11ea-9372-6266a4d4fac1.png">